src/lib/mira_integration/mira_integration_timer.c

```diff
@@ -41,7 +48,7 @@ void SysTick_Handler(
     previous_tick_time += diff * cyclesPerTick;
 
     miramesh_timer_schedule(
-        previous_tick_time + cyclesPerTick,
+        previous_tick_time + 1 * cyclesPerTick, /* Schedule for next tick */
```

Comment:
Detail, no need to change. But does the times 1 do anything functional? It's not making anything clearer for me, I'm just more confused.

Reply:
Yeah, may be it doesn't add much.

Initially I had:
`previous_tick_time + (TickType_t) 1 * cyclesPerTick,`

Then I removed the typecasting and added a comment. Now, I feel it's better remove the multiplication altogether and just have the comment.

Reply:
Done.